doc/56_Search.md

```diff
@@ -74,8 +74,8 @@ Selecting a file will open it an an editor.
 
 ## Open Workspace Symbol
 
-Press <kbd>Ctrl+O</kbd> (<kbd>⌘+O</kbd> on macOS) to find symbols in the workspace. The meaning
-of a symbol and what symbols are available is language-specific.
+Press <kbd>Ctrl+O</kbd> (<kbd>⌘+O</kbd> on macOS) to find symbols in the workspace. Note: what is
+considered a symbol is language-specific.
```

Comment:
How about `Note: what symbols are available is language specific` ... we're talking about finding symbols, so I care about the available symbols, not where they come from &lt;/nitpick&gt;

Reply:
you mean in the sense _what symbols are **supported**_?